application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCountDisplayRoutine.java

```diff
@@ -23,17 +24,16 @@ public class MemberCountDisplayRoutine implements Routine {
      */
     public MemberCountDisplayRoutine(Config config) {
         memberCountCategoryPredicate =
-                Pattern.compile(config.getMemberCountCategoryPattern() + "( - \\d+ Members)?")
+                Pattern.compile(config.getMemberCountCategoryPattern() + "( - [\\d,]+ Members)?")
```

Comment:
would be nice to write a comment for what this predicate would match as an example.

Reply:
![image](https://github.com/Together-Java/TJ-Bot/assets/73871477/aaa5c921-3dd8-45b7-ac80-cccb0d014876)

Reply:
i meant inside code itself 
//matches likes of `Info-1,000 Members`
or something like that

Reply:
nah

Reply:
I can do it if it's a bother, problem with regex is it looks like magical stuff so you have to either use a third party tool like a platform to validate it which you don't need to if a comment mentions what sort of pattern we're trying to match which can ofc save everyone a bit of time and make it more readable.